tests/ui/higher-ranked/trait-bounds/future.rs

```diff
@@ -0,0 +1,30 @@
+// edition:2021
+// revisions: classic next
+//[next] compile-flags: -Ztrait-solver=next
+//[next] check-pass
```

Comment:
You need to add something like:

```
//[classic] known-bug: .....
//[classic] failure-status: 101
//[classic] normalize-stderr-test "note: .*\n\n" -> ""
//[classic] normalize-stderr-test "thread 'rustc' panicked.*\n" -> ""
//[classic] normalize-stderr-test "(error: internal compiler error: [^:]+):\d+:\d+: " -> "$1:LL:CC: "
//[classic] rustc-env:RUST_BACKTRACE=0
```

where `known_bug: .....` is filled with an issue number.

Also, there is probably a stderr file that's missing from this PR.